src/ripple/app/hook/applyHook.h

```diff
@@ -32,6 +32,7 @@ class HookStateMap : public std::map<
                          std::tuple<
                              int64_t,  // remaining available ownercount
                              int64_t,  // total namespace count
+                             int16_t,  // hook state scale
```

Comment:
I don't mind if you use int16_t or uint16_t or uint32_t but keep it consistent with the function calls? like `maxHookStateDataSize(uint32_t hookStateScale)` same for the sf type